src/_pytest/terminal.py

```diff
@@ -734,6 +734,8 @@ def summary_warnings(self):
             else:
                 warnings = all_warnings
             self._already_displayed_warnings = len(warnings)
+            if not warnings:
```

Comment:
this is a good start, thanks for the swift fix- it should additionally be decided on when to print the documentation note (it seems its possible that it is indeed displayed twice even with the current setup)